packages/block-editor/src/store/private-actions.js

```diff
@@ -131,27 +131,21 @@ export const privateRemoveBlocks =
 			// corresponding to "important" blocks, i.e. blocks that require a
 			// removal prompt.
 			const queue = [ ...clientIds ];
-			let messageType = 'templates';
 			while ( queue.length ) {
 				const clientId = queue.shift();
 				const blockName = select.getBlockName( clientId );
 				if ( rules[ blockName ] ) {
 					blockNamesForPrompt.add( blockName );
 				}
 
-				if ( rules[ 'bindings/core/pattern-overrides' ] ) {
-					const blockAttributes =
-						select.getBlockAttributes( clientId );
-					if (
-						blockAttributes?.metadata?.bindings &&
-						JSON.stringify(
-							blockAttributes.metadata.bindings
-						).includes( 'core/pattern-overrides' )
-					) {
-						blockNamesForPrompt.add( blockName );
-						messageType = 'patternOverrides';
+				Object.values( rules ).forEach( ( rule ) => {
+					if ( rule.callBack ) {
```

Comment:
I believe in most of our APIs we used `callback` instead of `callBack`

Reply:
true, thanks, will fix that